spec/features/single_content_item_spec.rb

```diff
@@ -86,15 +86,15 @@
       end
 
       it 'renders context for page searches' do
-        expect(page).to have_selector '.searches .app-c-glance-metric__context', text: '0.4%'
+        expect(page).to have_selector '.searches .app-c-glance-metric__context', text: '4.0%'
```

Comment:
Should this be testing for two decimal places as per the design?

Reply:
Changed the data to something with 2 decimal places. Is this what you meant @theseanything